### PR TITLE
Document new processes for repo-management & IAM

### PR DESCRIPTION
**Pull request by mrnugget**
This documents the new grooming, retro, and planning processes for both teams.

This is the bare minimum. I expect us to change it in the future, which is also why it's duplicated between both teams. I expect each team to tune the processes to their requirements.

### CODE REVIEW

**Review by 0xnmn — APPROVED** [review]

File: content/departments/engineering/teams/iam/index.md [hunk 1]
```diff
@@ -62,26 +62,102 @@ The IAM team works alongside several other teams at Sourcegraph. You can find mo
...
+- _28min_: talk through items, start with the highest votes
+  - 5min per item, use a timer, be strict
+  - Once 5min are up: ask if 2 more minutes
+  - If possible: collect action items
 
-### Planning, execution, and issue tracking
+### Grooming
 
-The IAM team plans work based on our [long-term roadmap](https://docs.google.com/document/d/1XNrbBtkS8_lsjKxV8zvNfb1sn1Ug9Zhc24LFLCOa-Ic/edit?usp=sharing) and setting [quarterly goals](#goals-and-roadmap). During the quarter, we follow a flavor of the SCRUM process with biweekly sprints. Our cycle starts every second Tuesday with a retrospective, sprint review, and planning meetings. We set goals for each sprint and focus team efforts during the iteration on achieving these goals rather than closing a number of issues. It’s the outcomes and delivered customer value, not the output, that matters.
+**Goal**: The goal for the grooming session is to as many items in our backlog
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/iam/index.md]: the verb is missing, probably this:
```suggestion
**Goal**: The goal for the grooming session is to get as many items in our backlog
```

File: content/departments/engineering/teams/iam/index.md [hunk 2]
```diff
@@ -62,26 +62,102 @@ The IAM team works alongside several other teams at Sourcegraph. You can find mo
...
+_prioritized_ and _estimated_. We estimate together enough tickets so that we’re
+all on the same page on how to implement something and how long it will take, so
+that we can plan it in future sprint
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/iam/index.md]:
```suggestion
that we can plan it in future sprint.
```

File: content/departments/engineering/teams/iam/index.md [hunk 3]
```diff
@@ -62,26 +62,102 @@ The IAM team works alongside several other teams at Sourcegraph. You can find mo
...
 
-We are using [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) as our project tracking tool. The [list view of backlog](https://github.com/orgs/sourcegraph/projects/259/views/10) is publicly available in our team project.
+Requirements for the meeting:
 
-#### How we use GitHub Projects
+- DRI for a project prepares by creating tickets as much as possible
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/iam/index.md]:
```suggestion
- DRI for a project prepares by creating as many tickets as possible
```

File: content/departments/engineering/teams/iam/index.md [hunk 4]
```diff
@@ -62,26 +62,102 @@ The IAM team works alongside several other teams at Sourcegraph. You can find mo
...
+  - Are there things left-over from the last iteration? If so, why?
+- **Planning next 2 weeks**
+  - **Step 1: gather input**:
+    - What does Product need us to do in the next 2 weeks?
+      - _Ideally_ these requests are prioritized and “ready to be implemented”
+    - Are there any customer issues we need to work on?
+    - Other requests from outside?
+  - **Step 2: plan**:
+    - What do we want to work on?
+      - Go through each person, define what they'll work on, what they commit do
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/iam/index.md]:
```suggestion
      - Go through each person, define what they'll work on, what they commit to do
```

File: content/departments/engineering/teams/repo-management/processes.md [hunk 5]
```diff
@@ -43,24 +43,96 @@ More involved support requests follow the below process:
...
+- _5min_: celebrate what went well (read out all items)
+- _2min_: vote on items
+- _28min_: talk through items, start with the highest votes
+  - 5min per item, use a timer, be strict
+  - Once 5min are up: ask if 2 more minutes
+  - If possible: collect action items
+
+### Grooming
+
+**Goal**: The goal for the grooming session is to as many items in our backlog
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/processes.md]:
```suggestion
**Goal**: The goal for the grooming session is to get as many items in our backlog
```

File: content/departments/engineering/teams/repo-management/processes.md [hunk 6]
```diff
@@ -43,24 +43,96 @@ More involved support requests follow the below process:
...
+_prioritized_ and _estimated_. We estimate together enough tickets so that we’re
+all on the same page on how to implement something and how long it will take, so
+that we can plan it in future sprint
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/processes.md]:
```suggestion
that we can plan it in future sprint.
```

File: content/departments/engineering/teams/repo-management/processes.md [hunk 7]
```diff
@@ -43,24 +43,96 @@ More involved support requests follow the below process:
...
+
+Requirements for the meeting:
+
+- DRI for a project prepares by creating tickets as much as possible
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/processes.md]:
```suggestion
- DRI for a project prepares by creating as many tickets as possible
```

File: content/departments/engineering/teams/repo-management/processes.md [hunk 8]
```diff
@@ -43,24 +43,96 @@ More involved support requests follow the below process:
...
+  - Are there things left-over from the last iteration? If so, why?
+- **Planning next 2 weeks**
+  - **Step 1: gather input**:
+    - What does Product need us to do in the next 2 weeks?
+      - _Ideally_ these requests are prioritized and “ready to be implemented”
+    - Are there any customer issues we need to work on?
+    - Other requests from outside?
+  - **Step 2: plan**:
+    - What do we want to work on?
+      - Go through each person, define what they'll work on, what they commit do
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/processes.md]:
```suggestion
      - Go through each person, define what they'll work on, what they commit to do
```

**Review by sashaostrikov — COMMENTED** [review]
LGTM! Added some comments.

File: content/departments/engineering/teams/repo-management/processes.md (hunk 6 above)
Comment (sashaostrikov) [reply content/departments/engineering/teams/repo-management/processes.md]: one little full stop missed :)

**Review by sashaostrikov — APPROVED** [review]
one `.` is missed, apart from that: :shipit:

**Review by ryanslade — APPROVED** [review]

**Pull request merged**